### ISSUE TITLE
Documentation shows incorrect pre-commit configuration when using pyproject.toml

### ISSUE THREAD

**Issue by balazser** [PyCQA/docformatter#172]
I am trying to use the `docformatter` hook with `pre-commit` to format my Python code. However, I am encountering an issue where the `docformatter` hook does not respect the configuration in my `pyproject.toml` file when run through `pre-commit`, but it does respect the configuration when run manually with `poetry`.

Specifically, I have the following configuration for the `docformatter` hook in my `.pre-commit-config.yaml` file:

bash

```bash
- repo: https://github.com/PyCQA/docformatter
  rev: v1.5.0
  hooks:
    - id: docformatter
      args: ["--in-place", "--config", "./pyproject.toml"]
```

However, when I run `pre-commit run --all-files docformatter`, `docformatter` seems to ignore the configuration in `pyproject.toml` and uses its default settings instead. On the other hand, when I run `poetry run docformatter .`, `docformatter` correctly uses the settings in `pyproject.toml`.

Can someone explain why this might be happening, and how I can get `docformatter` to respect my `pyproject.toml` configuration when run through `pre-commit`?

Thank you.

**Comment by weibullguy** [comment]
This is similar to issue #169.  In both cases, I see you're using docformatter-1.5.0.  Could you try with docformatter-1.6.0 to see if the problem persists?  Also, is it ignoring all settings in pyproject.toml or only certain settings similar to #169?

**Comment by weibullguy** [comment]
Actually, now that I look at my `.pre-commit-config.yaml`
```
- repo: https://github.com/PyCQA/docformatter
  rev: v1.6.0
  hooks:
    - id: docformatter
      additional_dependencies: [tomli]
      args: [--in-place, --config, ./pyproject.toml]
```
you probably need to add the `additional_dependencies: [tomli]` line to your `.pre-commit-config.yaml`.

**Comment by balazser** [comment]
Hi @weibullguy,
I can confirm adding `additional_dependencies: [tomli]` solved my problem using `docformatter-1.5.0`.  I appreciate your help!
Should we mention it in the [doc here](https://docformatter.readthedocs.io/en/latest/usage.html#use-with-pre-commit)? 🤔
